tests/components/hassio/test_init.py

```diff
@@ -346,3 +357,63 @@ async def test_service_calls_core(hassio_env, hass, aioclient_mock):
         assert mock_check_config.called
 
     assert aioclient_mock.call_count == 5
+
+
+async def test_websocket_supervisor_event(
+    hassio_env, hass: HomeAssistant, hass_ws_client
+):
+    """Test Supervisor websocket event."""
+    assert await async_setup_component(hass, "hassio", {})
+    websocket_client = await hass_ws_client(hass)
+
+    events = []
+
+    def listener(ev):
+        events.append(ev.data)
+
+    assert hass.bus.async_listen(EVENT_SUPERVISOR_EVENT, listener)
```

Comment:
We have `async_capture_events` test helper:
https://github.com/home-assistant/core/blob/9917bb76fbaed50d57910a167f11ffe1b465cee6/tests/common.py#L1110-L1120